packages/service-library/src/servicelib/fastapi/prometheus_instrumentation.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=protected-access
+
+
+from fastapi import FastAPI
+from prometheus_fastapi_instrumentator import Instrumentator
+
+
+def instrument_app(app: FastAPI):
```

Comment:
I would rename this as `setup_prometheus_instrumentation(app: FastAPI)`

Reply:
fixed